packages/libs/react-components/src/styles/stitches.config.ts

```diff
@@ -169,4 +173,6 @@ export const darkTheme = createTheme('darkTheme', {
   },
 });
 
+export type IThemeColors = `$${keyof typeof theme.colors}`;
```

Comment:
Would it be worth using ScaleValue here instead of composing it ourselves?

Reply:
for now we dont need this import anymore.
will remove it